src/cljc/matcher_combinators/utils.cljc

```diff
@@ -1,9 +1,17 @@
-(ns matcher-combinators.utils)
+(ns matcher-combinators.utils
+  "Internal use only. Subject (and likely) to change.")
 
-(defn roughly? [expected actual delta]
-  (and (number? actual)
-       (>= expected (- actual delta))
-       (<= expected (+ actual delta))))
+(defn processable-number? [v]
```

Comment:
I'm not sure if we should hide this exception. If I'm trying to match a number against something invalid I would expect an exception.

Reply:
This will result assertion failure (a mismatch), but not an exception.

We were already returning false for non-numeric values. This just treats ##Inf, ##-Inf, and ##NaN as non-numeric for the purposes of this matcher. If the code produces ##Inf, that will show up in the failed assertion, which is sufficient info IMO. WDYT?

Reply:
OK, makes sense now